src/validator.ts

```diff
@@ -18,6 +18,11 @@ export class Validator {
   }
 
   validate(node: NodeToProcess, natspec: Natspec): string[] {
+    // Ignore fallback and receive
+    if (node instanceof FunctionDefinition && (node.kind === 'receive' || node.kind === 'fallback')) {
```

Comment:
I'd suggest making a helper here that can be used to check for `receive`, `fallback` or `constructor`
```solidity
if (matchesFunctionKind(node, 'receive') || matchesFunctionKind(node, 'fallback')) return [];
```

And then a bit further into the code:
```solidity
if (matchesFunctionKind(node, 'constructor')) {
  ...
}
```